overlord/hookstate/ctlcmd/set.go

```diff
@@ -154,7 +154,9 @@ func (s *setCommand) setInterfaceSetting(context *hookstate.Context, plugOrSlot
 		}
 
 		var value interface{}
-		err := json.Unmarshal([]byte(parts[1]), &value)
+		dec := json.NewDecoder(strings.NewReader(parts[1]))
```

Comment:
Would it make sense to use the helper here? If not, maybe a small comment why not?